internal/repos/github_test.go

```diff
@@ -337,6 +337,12 @@ func TestGithubSource_makeRepo(t *testing.T) {
 				Url:                   "https://github.com",
 				RepositoryPathPattern: "gh/{nameWithOwner}",
 			},
+		}, {
+			name: "name-with-owner",
+			schema: &schema.GitHubConnection{
+				Url:                   "https://github.com",
+				RepositoryPathPattern: "{nameWithowner}",
```

Comment:
Shouldn't this be `Owner`?

```suggestion
				RepositoryPathPattern: "{nameWithOwner}",
```